### PR TITLE
Fixed getDayOne()

### LINKED ISSUE #4: [ INVALID ] getDayOne()
Hi, when I was working on the Tests on the example/example1.dart (issue #2) I found out that the function getDayOne() returns:

```dart
    List<Covid19CountryAllStatus> list = List();

    for (var i in data) {
      list.add(
        Covid19CountryAllStatus(
          country: i['Country'],
          countryCode: i['CountryCode'],
          province: i['Province'],
          city: i['City'],
          cityCode: i['CityCode'],
          lat: i['Lat'],
          lon: i['Lon'],
          confirmed: i['Confirmed'],
          deaths: i['Deaths'],
          recovered: i['Recovered'],
          active: i['Active'],
          date: i['Date'],
        ),
      );
    }
    return list;
```

However, when we look at the API that this function is calling, it looks like this:

```json
[{"Country":"Canada","CountryCode":"CA","Province":"Ontario","City":"","CityCode":"","Lat":"51.25","Lon":"-85.32","Cases":1,"Status":"confirmed","Date":"2020-01-26T00:00:00Z"},
```
As you can see, we need to take in the list of:
 - country
 - countryCode
 - province
 - city
 - cityCode
 - lat
 - lon
 - **cases**
 - **status**
 - date

In your current list, the users might call for "deaths", "recovered", or "active" and this will result to a null because they do not exist in the API. 

image
the test above results to:
image

I want to fix this, please assign this to me. thank you

### PR DESCRIPTION
**Pull request by NesaByte**
fixes #4

### CODE REVIEW

File: lib/src/covid19client.dart [hunk 1]
```diff
@@ -51,29 +51,27 @@ class Covid19Client {
   /// It returns a list of [Covid19CountryAllStatus].
   ///
   /// The [status] can be: 'confirmed', 'recovered' or 'deaths'.
-  Future<List<Covid19CountryAllStatus>> getDayOne(
+  Future<List> getDayOne(
```
Comment (manu-urba) [inline lib/src/covid19client.dart]: Could you please specify the type of the elements in the list?
```dart
Future<List<Covid19CountryEx>> getDayOne(
```

File: lib/src/covid19client.dart [hunk 2]
```diff
@@ -51,29 +51,27 @@ class Covid19Client {
   /// It returns a list of [Covid19CountryAllStatus].
```
Comment (manu-urba) [inline lib/src/covid19client.dart]: Could you please change [Covid19CountryAllStatus] with [Covid19CountryEx]?
Thank you!

**Review by manu-urba — CHANGES_REQUESTED** [review]
Hello, thank you for your contribution.
I added 2 comments on your code, please check them out!

File: lib/src/covid19client.dart (hunk 1 above)
Comment (NesaByte) [reply lib/src/covid19client.dart]: aye, I'm on it!

**Comment by NesaByte** [comment]
Thank you for guiding me. I did the correct changes.

**Comment by manu-urba** [comment]
Al looks perfect!

**Comment by manu-urba** [comment]
Thank you!

**Pull request merged**